core/downloadfromdcthread.cpp

```diff
@@ -49,21 +50,38 @@ void DownloadThread::run()
 	qDebug() << "Finishing the thread" << errorText << "dives downloaded" << downloadTable.nr;
 }
 
+static void fill_supported_mobile_list()
+{
+	mobileProductList["Heinrichs Weikamp"] =
+			QStringList({{"OSTC Sport"}, {"OSTC 2N"}, {"OSTC 3"},
+				     {"OSTC 3+"}, {"OSTC 4"}});
+	mobileProductList["Shearwater"] =
+			QStringList({{"Petrel"}, {"Petrel 2"}, {"Perdix"}});
+}
+
 void fill_computer_list()
 {
 	dc_iterator_t *iterator = NULL;
 	dc_descriptor_t *descriptor = NULL;
 	struct mydescriptor *mydescriptor;
 
-	QStringList computer;
+#if defined(SUBSURFACE_MOBILE)
```

Comment:
So this is oversimplifying. Because what's supported isn't dependent on whether this is Subsurface-mobile that you are building, it depends on which platform you are on.

Which means if you are on 
#if defined (Q_OS_ANDROID)
we should allow BT, soon BLE, and FTDI devices
#if define (Q_OS_IOS)
we should only allow BLE
and other wise even with mobile we should allow everything, right?